ibm/conns/config.go

```diff
@@ -1832,18 +1832,16 @@ func (c *Config) ClientSession() (interface{}, error) {
 	session.apigatewayAPI = apigatewayAPI
 
 	// POWER SYSTEMS Service
-	piURL := ContructEndpoint(c.Region, "power-iaas.cloud.ibm.com")
 	ibmPIOptions := &ibmpisession.IBMPIOptions{
 		Authenticator: authenticator,
 		Debug:         os.Getenv("TF_LOG") != "",
-		Region:        c.Region,
-		URL:           EnvFallBack([]string{"IBMCLOUD_PI_API_ENDPOINT"}, piURL),
+		URL:           EnvFallBack([]string{"IBMCLOUD_PI_API_ENDPOINT"}, ""),
```

Comment:
@yussufsh , Why do you want to make the url as empty when IBMCLOUD_PI_API_ENDPOINT doesn't exist?